example_docs/docs/sphinx_guide/custom_directives.rst

```diff
@@ -0,0 +1,51 @@
+=================
+Custom Directives
+=================
+
+customcalloutitem
+=================
+
+.. raw:: html
+
+   <div class="tutorials-callout-container">
+      <div class="row">
```

Comment:
FYI this is not included in the `customcalloutitem` because it's common to have multiple "items" in a group. This is the outer container code.